### ISSUE TITLE
Editor does not handle noop rebases

### ISSUE THREAD

**Issue by whymarrh** [MitMaro/git-interactive-rebase-tool#4]
For example:

```
16:06:55 project (some-branch>) $ git rebase -i HEAD some-branch
thread 'main' panicked at 'Invalid line noop', src/main.rs:121
note: Run with `RUST_BACKTRACE=1` for a backtrace.
Could not execute editor
```

**Comment by MitMaro** [comment]
I haven't seen a noop rebase file in years. Thanks for the report!

Just for reference the git command to reproduce is:

    git rebase -i master --onto master

**Comment by MitMaro** [comment]
Added #5 to fix this.

**Comment by MitMaro** [comment]
Fixed in #5
